### ISSUE TITLE
Cache Mopidy playlists

### ISSUE THREAD

**Issue by wezzynl** [jaedb/spotmop#60]
When a user doesn't log in to Spotify from the webinterface, Spotmop fetches the playlists from Mopidy. This however does not seem to be cached. When having loads of playlists this takes a rather large amount of time. If it would be cached locally, it would be a major speed-up. Would need some sort of refresh button so users can refresh the cache.

**Comment by jodal** [comment]
libspotify should cache the playlists for us.

Note that there is a huge performance difference between the old and new playlist API in Mopidy. Make sure you use the "new" playlist API: `mopidy.playlists.as_list()` and `mopidy.playlists.get_items()`. The `mopidy.playlists.get_playlists()` command was deprecated in Mopidy 1.0 and should not be used.

**Comment by jaedb** [comment]
@jodal Brilliant, thanks for the heads up. @wezzynl I've updated the implementation and have this in the pipeline for the next release.

One thing I noticed (hence the notification when opening the Playlists page) is the time Mopidy takes to fetch the playlists depends on the power of the computer Mopidy is running on. My setup at home runs on a RaspberryPi 1 B+ and this page can therefore take some time to load.
